README.md

```diff
@@ -27,7 +29,7 @@ The other available scripts are:
 - `yarn lint` to ensure your code is properly formatted;
 - `yarn flow` to ensure your code is respecting the types given to your vaiables;
 - `yarn test-ci` to ensure your code should pass the Continuous Integration (CI) tests;
-- `yarn relay` to get the latest schema of the [graphql server](packages/graphql) 
+- `yarn relay` to get the latest schema of the [graphql server](packages/graphql)
```

Comment:
does not yarn relay run the relay-compiler? 
yarn graphql to get the latest schema?

Reply:
I looked at it and it does both things, it generates new schema and then runs relay-compiler.
So It looks like we don't need the 'yarn schema' command.